R/importWaterML1.r

```diff
@@ -244,7 +244,12 @@ importWaterML1 <- function(obs_url,asDateTime=FALSE, tz=""){
           obsDF <- full_join(obsDF, valParentDF, by = c("dateTime","tz_cd"))
         }
       }else{
-        obsDF <- data.frame()
+        #need column names for joining later
+        obsDF <- data.frame(dateTime=character(0), tz_cd=character(0), stringsAsFactors = FALSE)
+        if(asDateTime){
+          obsDF$dateTime <- as.POSIXct(obsDF$dateTime)
```

Comment:
It would be set earlier when there is actual data.  This is just creating the columns so there isn't an error when you merge an empty data frame.  Would time zones still matter on an empty data frame?